src/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -401,7 +401,7 @@ export default class SummaryPanelNode extends React.Component<SummaryPanelPropTy
       return (
         <>
           <div>
-            <KialiIcon.Info /> Sparkline charts not supported for unknown node. Use edge for details.
+            {KialiIcon.Info()} Sparkline charts not supported for unknown node. Use edge for details.
```

Comment:
Why? It wasn't working with the tag syntax?

Reply:
Ahh I see, is because the style? right?

Reply:
I'd be happy if I didn't have to eval the KialiIcon but I didn't see a way to gave that work and have the style option.  Suggestions welcome...

Reply:
Thanks for the PR!